src/index.ts

```diff
@@ -12,6 +12,8 @@ export * as SiteSearch from './services/site-search/site-search.service'
 export * as UserManagement from './services/user-management/userManagement.service'
 export * as DataLayer from './services/dataLayer/dataLayer.service'
 export * as ErrorTracking from './services/error-tracking/errorTracking.service'
+export * as CrossDomain from './services/cross-domain/crossDomain.service'
```

Comment:
Suggestion: this clearly maps 1:1 from the docs and avoids ambiguity

```suggestion
export * as CrossDomainLinking from './services/cross-domain/crossDomainLinking.service'
```